src/main/java/bob/task/Deadlines.java

```diff
@@ -0,0 +1,56 @@
+package bob.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+import bob.exception.InvalidInputException;
+
+/**
+ * Tasks that need to be done before a specific date/time.
+ */
+public class Deadlines extends Task {
```

Comment:
```suggestion
public class Deadline extends Task {
```
Consider renaming the class from `Deadlines` to `Deadline` since each instance represents a single task with a specific deadline?